### PR TITLE
Update SBT to 1.2.8

### PR DESCRIPTION
**Pull request by kenoir**
Use faster/more recent version of SBT - remove deprecated usage of util.Json.

This should also fix the current issue with the Travis build.

### CODE REVIEW

File: bag_unpacker/src/main/resources/logback.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <configuration>
   <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
     <encoder>
-      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+      <pattern>%d{HH:mm:ss.SSS} %X{ingestId} %X{externalIdentifier} [%thread] %-5level %logger{36} - %msg%n</pattern>
```
Comment (jamesgorrie) [inline bag_unpacker/src/main/resources/logback.xml]: Where are you getting there from ?

Comment (kenoir) [reply bag_unpacker/src/main/resources/logback.xml]: leftover code

File: project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=0.13.17
+sbt.version=1.2.8
```
Comment (alexwlchan) [inline project/build.properties]: How did I not notice that 0.13.17 is older than 1.2.8?

🤦‍♀

Comment (kenoir) [reply project/build.properties]: I think the 0.13 branch is forked from the 1.x branch and still maintained, so it's not clear if it is actually older. The `0.13` fork is the "old" one though.

**Comment by alexwlchan** [comment]
Close enough.

**Pull request merged**